addons/area_markers/functions/fnc_isInEditMode.sqf

```diff
@@ -0,0 +1,21 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Checks if the Zeus display is in marker editing mode.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * In Marker Editing Mode <BOOL>
+ *
+ * Example:
+ * [] call zen_area_markers_fnc_isInEditMode
+ *
+ * Public: No
+ */
+
+visibleMap
+&& {!dialog}
```

Comment:
Simple bool checks are too cheap for lazy eval, I even tested it myself.
```suggestion
&& !dialog
```

Reply:
In ZEN, we always use lazy eval as a matter of style. The difference in performance, if any, is negligible.

Reply:
A selfish pure excuse, as invalid as a null check on a nil variable, as dangerous as going down the Niagara Falls in a barrel.